test/src/main/java/org/apache/accumulo/test/fate/FatePoolsWatcherITBase.java

```diff
@@ -171,12 +171,8 @@ protected void testIncrease1(FateStore<PoolResizeTestEnv> store, ServerContext s
       assertEquals(0, fate.getTxRunnersActive(set2));
       assertEquals(numWorkersSet3, fate.getTxRunnersActive(set3));
       assertEquals(numWorkersSet4, fate.getTxRunnersActive(set4));
-      allAssertsOccurred = true;
-    } catch (Exception e) {
-      System.out.println("Failure: " + e);
     } finally {
       fate.shutdown(30, TimeUnit.SECONDS);
-      assertTrue(allAssertsOccurred);
```

Comment:
Ahh good catch. I think my reason for introducing this allAssertsOccurred was because I was seeing the test pass when an assertion would fail and therefore the subsequent assertions would not occur. I also did not like this allAssertsOccurred hack, but I wasn't sure why the test was passing with an assertion failure.

Looking at this now, it's very obvious that this catch Exception was hiding the assertion failure. Not sure why I had this catch to begin with...

Thanks for this fix!